SCF/SCF-session3/Assignment1/StringOperations.java

```diff
@@ -0,0 +1,88 @@
+
+/**
+ * 
+ * @author Ramdev
+ * implements methods for comparing two stings , reversal of string, to case vice-versa and to find largest word from a string 
+ * 
+ */
+public class StringOperations 
+{
+	/**
+	 * compare two strings
+	 * @param first_string
+	 * @param second_string 
+	 * @return 1 if both strings are equal else 0
+	 */
+	public static int compareTwoStrings(String first_string, String second_string)
+	{
+		if(first_string.equals(second_string))
+			return 1;
+		else
+			return 0;
+	}
+	/**
+	 * reverse the string
+	 * @param string to be reversed
+	 * @return reversed string
+	 */
+	public static String revese(String string)
+	{
+		String reversed_string = "";
```

Comment:
use stringBuilder here, string instance are immutable which means everytime you change the string it creates a new instance with new value, string builder is an better alternative